service/api_client.py

```diff
@@ -1,44 +1,59 @@
 import requests  # type: ignore
 import config
+import logging
+import logging.config
 from datetime import datetime                                                                          # type: ignore
 
+LOGGER = logging.getLogger(__name__)
+
 LAND_REGISTRY_PAYMENT_INTERFACE_BASE_URI = config.CONFIG_DICT['LAND_REGISTRY_PAYMENT_INTERFACE_BASE_URI']
 REGISTER_TITLE_API_URL = config.CONFIG_DICT['REGISTER_TITLE_API'].rstrip('/')
 LAND_REGISTRY_PAYMENT_INTERFACE_URI = config.CONFIG_DICT['LAND_REGISTRY_PAYMENT_INTERFACE_URI']
 
 
 def get_title(title_number):
+    LOGGER.debug("STARTED: get_title title_number: {}".format(title_number))
     response = requests.get('{}/titles/{}'.format(REGISTER_TITLE_API_URL, title_number))
-
+    LOGGER.debug('get_title response: %s', response)
```

Comment:
This may be me not understanding python but isn't this the old way of formatting strings? Should we be consistent and use .format as we have in all the other debug messages?